.github/bot/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "auto-merge-eip",
+  "version": "1.0.0",
+  "description": "auto merge eip",
+  "main": "index.js",
+  "scripts": {
+    "test": "NODE_ENV=testing jest __tests__",
+    "build": "ncc build ./src/index.ts",
+    "watch": "ncc build ./src/index.ts --watch",
```

Comment:
What is `ncc` and why is it being used here instead of `npx tsc`?

Reply:
I dug into it a bit, and I don't think it is necessary.  We can simply do `npx tsc` for the build/watch scripts, and we don't actually need a single file output I don't believe.  If we *do* need a single file output for some reason, TypeScript has the ability by using `outFile` instead of `outDir`.

Reply:
cool that works!